frontend/mulighetsrommet-veileder-flate/src/components/modal/delemodal/DelMedBrukerContent.tsx

```diff
@@ -101,18 +101,25 @@ export function DelMedBrukerContent({
       )}
 
       {visPersonligMelding ? (
-        <Textarea
-          ref={personligHilsenRef}
-          className={delemodalStyles.personligHilsen}
-          size="medium"
-          value={state.hilsen}
-          label=""
-          hideLabel
-          onChange={redigerHilsen}
-          maxLength={MAKS_ANTALL_TEGN_HILSEN}
-          data-testid="textarea_hilsen"
-          error={handleError()}
-        />
+        <>
+          <Textarea
+            ref={personligHilsenRef}
+            className={delemodalStyles.personligHilsen}
+            size="medium"
+            value={state.hilsen}
+            label=""
+            hideLabel
+            onChange={redigerHilsen}
+            maxLength={MAKS_ANTALL_TEGN_HILSEN}
+            data-testid="textarea_hilsen"
+            error={handleError()}
+          />
+          <p>
```

Comment:
Burde bruke `<BodyShort/>` istedenfor `<p/>` her for å få alle tekster like

Reply:
Et annet spørsmål er om vi trenger å wrappe det i `<BodyShort/>` i det hele tatt, med tanke på at teksten uansett ligger i en `Alert`? Er det for å fjerne den blå bakgrunnsfargen som hører til Alerten?

Reply:
Nei, wrappet i p for å få litt luft rundt teksten, men kan teste bodyshort

Reply:
Bodyshort rendrer til en div så jeg beholder p'en

Reply:
Burde vi bruke noe annet enn styling for å style det, da? Jeg mener, trenger vi å bruke p eller BodyShort eller whatevs for å få luft rundt teksten, eller burde vi heller style Alerten med margins?

Reply:
Godt spm. Føles jo litt hassle å skulle style alle disse greiene når vi bare kan bruke standard html-tags

Reply:
Jeg foretrekker å heller style komponentene enn å bruke masse html-tags overalt, da det fort kan bli uoversiktlig 🤓😅

Reply:
Semantisk korrekte html-tags er jo bedre å bruke enn div's da det blir enklere for blant annet skjermlesere å forstå det semantiske innholdet